src/multitoken/category_registry.cairo

```diff
@@ -49,13 +49,15 @@ pub mod MultiTokenCategoryRegistry {
     component!(path: SRC5Component, storage: src5, event: SRC5Event);
 
     #[abi(embed_v0)]
-    impl OwnableMixinImpl = OwnableComponent::OwnableMixinImpl<ContractState>;
+    impl OwnableTwoStepImpl = OwnableComponent::OwnableTwoStepImpl<ContractState>;
```

Comment:
Should this be Mixin?, in `pwn_config` it was not, so I kept this way. Let me know if it needs to be mixin

Reply:
2steps is good